mcstatus/tests/test_pinger.py

```diff
@@ -28,11 +29,13 @@ def test_read_status(self):
         )
         status = self.pinger.read_status()
 
-        assert status.raw == {
-            "description": "A Minecraft Server",
-            "players": {"max": 20, "online": 0},
-            "version": {"name": "1.8-pre1", "protocol": 44},
-        }
+        assert status == JavaServerResponse(
+            players=JavaServerPlayers(max=20, online=0, list=None),
+            version=JavaServerVersion(name="1.8-pre1", protocol=44),
+            motd="A Minecraft Server",
+            latency=status.latency,
+            icon=None,
+        )
```

Comment:
Same issue as with the async version, we should probably patch `JavaServerResponse`'s `__init__` and only check the raw data, we're not testing the parsing and the response classes here, there should be a dedicated test for those if any.

Reply:
Fixed in dda07c207c9acc3ebcab2755bdbbfa9bfc91f323.